README.md

```diff
@@ -2,44 +2,203 @@
+  maintained in the [arrow][arrow-repo] repository.
+- [Crossbow][crossbow-readme] for binary
+  packaging and nightly tests. Crossbow provides a command line interface to
+  programatically trigger Travis and Appveyor builds by creating git branches
+  in [ursalabs/crossbow][crossbow-repo] repository.
+  It is maintained within [arrow][arrow-repo], for more see its
+  [guide][crossbow-readme].
+- [Ursabot][ursabot-repo] implemented on top of [buildbot][buildbot-docs] CI
+  framework, hosted at [ci.ursalabs.org][ursabot-url] on Ursalabs' servers.
+  It provides:
+
+    - A set of builds testing the C++ implementation and Python bindings on
+      multiple operating systems and architectures (most of these are executed
+      automatically on each commit and pull request).
+    - On-demand builds requested by commenting on a pull request.
+    - On-demand benchmarks requested by commenting on a pull request, the
+      benchmark results are reported as github comments. The recently developed
+      [Archery][archery-readme] command-line tool is responsible for running
+      and comparing the benchmarks.
+    - Special builds for triggering other systems' builds, like
+      [crossbow][crossbow-readme]'s packaging tasks by commenting on a pull
+      request.
+
+ <!-- Comprehensive, automatic nightly checks allow us to catch quickly any
+regressions that our human expertise missed so that we can fix them right away.
+(Call it "continual integration" rather than "continuous integration".) -->
+
+
+## Driving Ursabot
+
+Allowing PR reviewers to request additional checks on demand within the review
+process makes it easier for us to apply extra scrutiny at review time while
+also conserving CI bandwidth by using human expertise to know which checks are
+needed.
+
+### via Comments
+
+Ursabot receives github events through a webhook. It listens on pull request
+comments mentioning @ursabot. It follows the semantics of a command line
+interface, to see the available commands add a comment on the pull request: `@ursabot --help`.
+
+The @ursabot GitHub user will respond or [react][github-reactions] that it has
+started a build for you. Unfortunately, it does not currently report back
+on the build status. The reporters are already implemented. They will be
+enabled once the proper github integration permissions are set for the
+[apache/arrow][arrow-repo] repository. Until that you have to search around the
+[buildbot UI][ursabot-url] for it. The command parser is implemented in [commands.py](commands.py).
+
+Currently available commands:
+
+  - `@ursabot build`: Triggers all the ursabot tests. These tests are run
+    automatically, but this is a convinient way to force a re-build.
+  - `@ursabot benchmark`: Triggers C++ benchmarks and sends back the results as
+    a github comment and highlights the regressions.
+  - `@ursabot crossbow test cpp-python`: Triggers the `cpp-python` test group
```

Comment:
Is `cpp-python` the only thing that goes with `test`? Do we expect there to be more? If there are more, will `crossbow test` do them all? What does `@ursabot crossbow test cpp-python` run that `@ursabot build` does not?

Reply:
The builder which trigger crossbow builds. There are multiple test groups defined in crossbow, see the referenced tests.yml